### PR TITLE
tests: enhance ext.config.rpm-ostree-countme test

### PR DESCRIPTION
**Pull request by jlebon**
Instead of hardcoding a list of the same string for different values of
N, just teach it to look for N/N for any value.

### CODE REVIEW

File: tests/kola/rpm-ostree-countme/test.sh [hunk 1]
```diff
@@ -53,16 +53,21 @@ for i in $(seq 1 5); do
...
 	# Check rpm-ostree count me output
 	output="$(journal_after_cursor)"
-	# Depending on the stream, we expect different numbers of countme-enabled repos
-	if [[ "${output}" != "Successful requests: 1/1" ]] && \
-	   [[ "${output}" != "Successful requests: 2/2" ]] && \
-	   [[ "${output}" != "Successful requests: 3/3" ]] && \
-	   [[ "${output}" != "Successful requests: 4/4" ]]; then
+	trimmed=${output##Successful requests: }
+	if [[ ! $trimmed =~ ^[0-9]+/[0-9]+$ ]]; then
 		echo "rpm-ostree-countme service ouput does not match expected sucess output (try: $i):"
```
Comment (bgilbert) [inline tests/kola/rpm-ostree-countme/test.sh]: `output`/`success` while we're here?

File: tests/kola/rpm-ostree-countme/test.sh [hunk 2]
```diff
@@ -53,16 +53,21 @@ for i in $(seq 1 5); do
...
 		echo "${output}"
 		sleep 10
 		continue
 	fi
+	tries=${trimmed%%/*}
+	total=${trimmed##*/}
+	if [ "${tries}" != "${total}" ]; then
+		echo "rpm-ostree-countme service ouput shows failed requests (try: $i):"
```
Comment (bgilbert) [inline tests/kola/rpm-ostree-countme/test.sh]: `output`

**Review by cgwalters — DISMISSED** [review]

**Review by bgilbert — DISMISSED** [review]

File: tests/kola/rpm-ostree-countme/test.sh (hunk 1 above)
Comment (jlebon) [reply tests/kola/rpm-ostree-countme/test.sh]: Hmm, we print the full output already, no?

File: tests/kola/rpm-ostree-countme/test.sh (hunk 2 above)
Comment (jlebon) [reply tests/kola/rpm-ostree-countme/test.sh]: Not sure I follow this one. We already print the full `output`, and here we're printing the try we're at in the overall loop.

Comment (bgilbert) [reply tests/kola/rpm-ostree-countme/test.sh]: Well, we print the full `ouput`.  :stuck_out_tongue:

File: tests/kola/rpm-ostree-countme/test.sh (hunk 1 above)
Comment (jlebon) [reply tests/kola/rpm-ostree-countme/test.sh]: Ahhh, misinterpreted the "while we're here" bit. Typos fixed!

**Review by bgilbert — APPROVED** [review]

**Pull request merged**